exchanges/throw-on-error/package.json

```diff
@@ -49,15 +49,15 @@
   },
   "devDependencies": {
     "@urql/core": "workspace:*",
-    "graphql": "^16.0.0"
+    "graphql": "^16.0.0",
+    "wonka": "^6.3.2"
   },
   "peerDependencies": {
     "@urql/core": "^5.0.0"
   },
   "dependencies": {
     "@urql/core": "workspace:^5.1.1",
-    "graphql-toe": "0.1.2",
-    "wonka": "^6.3.2"
```

Comment:
In other exchanges we do keep this in dependencies, I would prefer not to deviate from that

Reply:
Addressed :+1: